addons/web-support/impl/src/main/java/org/jboss/windup/web/addons/websupport/services/WindupExecutorServiceImpl.java

```diff
@@ -80,13 +81,24 @@ public void execute(WindupProgressMonitor progressMonitor, Collection<Path> rule
             configuration.setOptionValue(SourceOption.NAME, Collections.singletonList(source));
 
         if (targets != null && targets.size() > 0) {
+            Object targetFromOptions = otherOptions.get(TargetOption.NAME);
```

Comment:
Won't this fix only work for targets and not for other options that might potentially need to be treated as a list?

Reply:
I was thinking about the case of multiple targets from advanced options that will become a list of targets

Reply:
Yes, but it could also happen with source. The fix for master seems to work generically, right?

Reply:
Oh, I misunderstood the context of this PR. It isn't necessary at all from what I can tell? I think multiple targets worked even without it? Maybe you can demo at the standup as I could be missing something else too.

Reply:
yeah, sure: i'll show you @ standup.
In short, this [`setOptionValue`](https://github.com/windup/windup-web/blob/master/addons/web-support/impl/src/main/java/org/jboss/windup/web/addons/websupport/services/WindupExecutorServiceImpl.java#L91) w/ key `target` (from the target advanced option) overrides the previous `configuration.setOptionValue(TargetOption.NAME, targets);` since [WindupConfiguration.configurationOptions](https://github.com/windup/windup/blob/master/exec/api/src/main/java/org/jboss/windup/exec/configuration/WindupConfiguration.java#L52) field is a `Map`

Reply:
Yes, I tried it again and I understand now. I'm going to try a slightly more generic fix, though, and maybe you can try that?